app/javascript/tests/history/History.test.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react'
+import { shallow } from 'enzyme';
+import { Button } from 'react-bootstrap';
+import _ from 'lodash';
+import History from '../../components/history/History.js'
+import { mockHistory1, mockHistory2 } from '../mocks/mockHistories'
+
+const authyToken = 'Q1z4yZXLdN+tZod6dBSIlMbZ3yWAUFdY44U06QWffEP76nx1WGMHIz8rYxEUZsl9sspS3ePF2ZNmSue8wFpJGg==';
+
+beforeEach(() => {
+  window.BASE_PATH = ""
+})
+
+describe('History', () => {
+  it('Properly renders non-comment histories', () => {
+    const wrapper = shallow(<History key={mockHistory1.id} history={mockHistory1} authenticity_token={authyToken} />);
+    const divEditId = "edit-history-item-" + mockHistory1.id;
+    const divDeleteId = "delete-history-item-" + mockHistory1.id;
+    expect(wrapper.find('b').exists()).toBeTruthy();
+    expect(wrapper.find('b').text()).toEqual(mockHistory1.created_by);
+    expect(wrapper.find('.h5').find('span').text()).toEqual(mockHistory1.history_type);
+    expect(wrapper.find('#' + divEditId).exists()).toBeFalsy();
```

Comment:
so this is actually testing to see if the tooltip is not present, when we want to test that the buttons . I would do a find Button and make sure the length is 0.  For the following test make sure the length is 2 and then test each button and make sure the icon is correct (trash and edit)

Reply:
fixed here: https://github.com/SaraAlert/SaraAlert/pull/793/commits/336f7f2565ad7b9798767f02dd44c3442b64135d